### DISCUSSION TITLE
Ability to prevent render running from lifecycle hooks

### DISCUSSION THREAD

**Discussion by kocoten1992** [livewire/livewire#2965] [3. Feature ideas]
Sometimes I use third party js library that auto generate many html element on the page.

When combine with livewire, it wipe out these custom element, it should be possible to prevent `render` from running from lifecycle hooks ?

**Comment by nuernbergerA** [comment]
Just use [wire:ignore](https://laravel-livewire.com/docs/2.x/alpine-js) and all children will not update on re-render

![image](https://user-images.githubusercontent.com/13331388/122044099-65d6b080-cddc-11eb-882b-3c72d4664e2d.png)

**Reply by kocoten1992** [reply]
@nuernbergerA thank you very much!!

I think this shouldn't be in alpine.js section alone, I only look many place in docs except this section -__ (many people also need this but not need alpine.js)

**Reply by nuernbergerA** [reply]
Indeed docs need some love. I bet V3 will be better documented. Anyhow we always accept PR to the docs. Feel free to contribute 😉
